static/js/aria-controls.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+(function() {
+
+    function accordionToggles(tabCtrl) {
+
+        function toggle (toggle, open) {
+          var button = toggle.getAttribute('aria-controls');
+          var panel = document.querySelector(button);
+
+          if (open) {
+              panel.setAttribute('aria-hidden', false);
+              toggle.setAttribute('aria-expanded', true);
+          } else {
+              panel.setAttribute('aria-hidden', true);
+              toggle.setAttribute('aria-expanded', false);
+          }
+        }
+
+        function toggleAll (event) {
+            var target = event.target;
+
+            // Filter through all toggles and toggle visiblity
+            toggles.filter(function (toggle) {
+                return toggle !== target;
+            }).forEach(toggle);
```

Comment:
Here you could do something that won't "navigate" through 2 arrays:
```js
toogles.forEach(function (element) {
  if (element !== target) {
    toogle(element):
  }
});
```

I think you can find a more elegant name for `element`